system/include/emscripten/heap.h

```diff
@@ -18,7 +18,7 @@ extern "C" {
 
 // Returns a pointer to a memory location that contains the heap DYNAMICTOP
 // variable (the end of the dynamic memory region)
-intptr_t *emscripten_get_sbrk_ptr(void) EM_IMPORT(emscripten_get_sbrk_ptr);
+intptr_t *emscripten_get_sbrk_ptr(void);
```

Comment:
Presumably we can't completely remove this functions as a followup?

Reply:
I think so, but it's declared in `./system/include/emscripten/heap.h`, and I'm not sure offhand why.

Reply:
It is but I think it was only added in the last few months so hopefully is safe to cull :-/

Reply:
Probably, yeah. Maybe as a followup though?

Reply:
Oh yes, thats what I originally suggested.